presto-docs/src/main/sphinx/functions/conversion.rst

```diff
@@ -32,3 +32,11 @@ Miscellaneous
         SELECT typeof(123); -- integer
         SELECT typeof('cat'); -- varchar(3)
         SELECT typeof(cos(2) + 1.5); -- double
+
+.. function:: parse_data_size(varchar) -> double
```

Comment:
I'm not sure if the "conversion" page is the best place for this function. I just didn't find any better, and wasn't sure if it is a good idea to add a separate page just for a single function